lib/backburner.js

```diff
@@ -30,6 +30,16 @@ export default function Backburner(queueNames, options) {
     end: [],
     begin: []
   };
+
+  var backburner = this;
+  this.executeTimers = function () {
```

Comment:
are you ok with moving the content of this function to on the prototype?

```js
prototype.executeTimers = function () {
  .. content of your function.
}
```

We can then in the constructor do what you just did.

```js
var backburner = this;
this.boundExecuteTimers = function() {
  return backburner.executeTimers.bind(backburner);
}
```

Why?

* the function logic is where one would expect it to be.